universal_transfer_operator/src/universal_transfer_operator/integrations/fivetran.py

```diff
@@ -0,0 +1,292 @@
+class Group:
+    """
+    Fivetran group details
+    :param name: The name of the group within Fivetran account.
+    """
+
+    name: str
+
+
+@attr.define
+class Destination:
+    """
+    Fivetran destination details
+
+    :param service: services as per https://fivetran.com/docs/rest-api/destinations/config
+    :param region: Data processing location. This is where Fivetran will operate and run computation on data.
+    :param time_zone_offset: Determines the time zone for the Fivetran sync schedule.
+    :param run_setup_tests: Specifies whether setup tests should be run automatically.
+    :param config: Configuration as per destination specified at https://fivetran.com/docs/rest-api/destinations/config
+    """
+
+    service: str
+    region: str | None
+    time_zone_offset: str | None
+    run_setup_tests: bool | None
+    config: dict
+
+
+@attr.define
+class Connector:
+    """
+    Fivetran connector details
+
+    :param connector_id: The unique identifier for the connector within the Fivetran system
+    :param service: services as per https://fivetran.com/docs/rest-api/destinations/config
+    :param config: Configuration as per destination specified at https://fivetran.com/docs/rest-api/connectors/config
+    :param paused: Specifies whether the connector is paused. Defaults to True.
+    :param pause_after_trial: Specifies whether the connector should be paused after the free trial period has ended.
+     Defaults to True.
+    :param sync_frequency: The connector sync frequency in minutes Enum: "5" "15" "30" "60" "120" "180" "360" "480"
+     "720" "1440". Default to "5"
+    :param daily_sync_time: Defines the sync start time when the sync frequency is already set or being set by the
+    current request to 1440.
+    :param schedule_type: Define the schedule type
+    :param connect_card_config: Connector card configuration
+    :param trust_certificates: Specifies whether we should trust the certificate automatically. The default value is
+     FALSE. If a certificate is not trusted automatically,
+    :param trust_fingerprints: Specifies whether we should trust the SSH fingerprint automatically. The default value
+    is FALSE.
+    :param run_setup_tests: Specifies whether the setup tests should be run automatically. The default value is TRUE.
+    """
+
+    connector_id: str | None
+    service: str
+    config: dict
+    connect_card_config: dict
+    paused: bool = True
+    pause_after_trial: bool = True
+    sync_frequency: str = "5"
+    daily_sync_time: str = "00:00"
+    schedule_type: str = ""
+    trust_certificates: bool = False
+    trust_fingerprints: bool = False
+    run_setup_tests: bool = True
+
+
+@attr.define
+class FiveTranOptions:
+    retry_limit: int = 3
+    retry_delay: int = 1
+    poll_interval: int = 15
+    schedule_type: str = "manual"
+    connector: Connector = attr.field(factory=Connector)
+
+
+class FivetranIntegration(TransferIntegration):
+    """
+    Class for FiveTran.
+    """
```

Comment:
Can we add a bit more documentation about what a Fivetran Integration is?

Reply:
[Done](https://github.com/astronomer/astro-sdk/pull/1492/commits/8943d29df9d6d565ce668ceb9c7497aa3775c959)